data/kahet/kahet missions.txt

```diff
@@ -738,3 +738,1169 @@ event "pacified ka'sei"
+				goto depart
+
+			label time
+			`	"Around four minutes," Dusk replies hurriedly. "But we will talk about that another time. We need to depart as soon as possible, all of us together. I do not know how much time we have." You hastily connect your helmet back to the life support and follow them out of the airlock, trying to keep up as best as you can.`
+			label depart
+			`	It takes only a few minutes to launch the <ship>, the six of you standing in the cockpit with trepidation. "We will take care of the necessary analysis while you bring us to orbit. Hopefully we have been sent to the Milky Way, and not to yet another unreachable galaxy."`
+				launch
+	on enter
+		conversation "end of Patir 2 - on enter"
+
+	on complete
+		payment 1570000
+		clear "remnant: dusk busy"
+		clear "remnant: dusk busy"
+		event "patir mystery 2 timer" 45
+		set "initial patir exploration"
+		conversation "end of Patir 2 - on complete"
+
+event "Patir returns"
+	system Patir
+		pos -38 553
+		haze _menu/haze-red
+		remove hazard "Beyond Patir Shield Eraser" 1
+		add hazard "Ember Waste Base Heat" 100
+		add hazard "Ember Waste Base Storm" 9000
+
+mission "Ka'het: Patir Mystery 13A"
+	name "Return to <planet>"
+	description "Now that the Patir black hole appears to have returned to the Milky Way, return to <planet> together with Dusk."
+	source "Builder Settlement"
+	destination "Aventine"
+	to offer
+		has "Ka'het: Patir Mystery 12A: done"
+	on offer
+		conversation
+			`The time has come: the walk toward the lab is quick and straightforward, and you cover the whole distance in a matter of minutes. Peeking through the hatch's window, you see the rest of the group preparing their backpacks, with prepacked toolboxes surrounding them in every direction. You wave through the glass, and they return the greeting.`
+			`	You move the airlock's heavy door to pass through, and wait for the room to be re-pressurized before walking in. By the time the hatch unlocks, however, the Remnant are gone, hastily leaving their bags on the floor.`
+			`	As you take a few steps inside the lab, you turn around to find them all glued to the windows with an awestruck gaze. The sky has again turned black, and a few dim lines are already starting to appear in the distance.`
+			`	While stars and galaxies zoom in the night sky, a few of the Remnant run around the lab, turning on various instruments. However, most of them remain with their faces stuck to the glass. The lights start converging toward a single point, forming a large, but rapidly shrinking mass in the background.`
+			`	The stars begin to slow down before settling into their final positions. After all the time you have spent here, it isn't difficult to recognize that the sky has changed again. Next to you, the rest of the team is already analyzing the data gathered; Dusk seems to be writing down something from a chronometer.`
+			action
+				event "Patir returns" 0
```

Comment:
> Derpy asked to put this here so that the map will show the right position before and after patir moves

Has this been tested? Since there's no choice prior to this point in the conversation, I'm fairly sure it will apply this event immediately upon the conversation appearing, which is the same as if it were placed directly in the trigger, outside of the conversation.
If you were to put the action after the choice, then it would only apply after that choice has been made, but Patir has already moved before we reach the choice.

Reply:
> Has this been tested?

No. Moved back outside of the conversation